README.md

```diff
@@ -75,6 +75,8 @@ The preferred method is to use the default AWS credentials pattern.  If no AWS c
       "baseUrlDirect": false, // default value
```

Comment:
Please add the new parameters `presignedUrl`, `presignedUrlExpires` to the [parameter table](https://github.com/parse-community/parse-server-s3-adapter/blob/master/README.md#parameters).